### PR TITLE
Update Danish Translation (85% / 400)

### PR DESCRIPTION
**Pull request by Stisen1**
- Focused translations of artifact prompts when interacting with them
- Some minor changes to some of the artifact titles.

### CODE REVIEW

File: files/lang/dk.po [hunk 1]
```diff
@@ -9483,6 +9643,10 @@ msgid ""
 "together is stunning. As you pull the scroll out, you feel imbued with "
 "magical power."
 msgstr ""
+"Du finder en udførlig beholder, som rummer en gammel pergamentrulle. Runerne "
+"på beholderen er meget gamle, og det kunstneriske, som det blev sat sammen "
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:9647]: Oh... Right... It has been changed to something less google.translaty

File: files/lang/dk.po [hunk 2]
```diff
@@ -9404,16 +9550,24 @@ msgid ""
 "original crystal grows the lost piece back. You decide to stuff the entire "
 "patch into a pouch and take it with you."
 msgstr ""
+"Du søger ly for en storm i en lille hule, bemærker du undervejs en lille "
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:9553]: Exacly. It has been added

File: files/lang/dk.po [hunk 3]
```diff
@@ -9337,6 +9468,9 @@ msgid ""
 "righteousness of your cause. The newly loyal subject gives you his endless "
 "pouch of sulfur to help with the war effort."
 msgstr ""
+"Du besøger en alkymist, som efter at have set din hær bliver påvirket af din "
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:9471]: yes

File: files/lang/dk.po [hunk 4]
```diff
@@ -9020,6 +9101,10 @@ msgid ""
 "them and teach them to make fire with sticks. Believing you to be a god, the "
 "apes give you their rock."
 msgstr ""
+"Dine vandringer bringer dig i kontakt med en stamme af abelignende væsener, "
+"der bruger en magisk lavasten, som aldrig køler, til at tænde deres ild med. "
+"Du forbarmer dig over dem og lærer dem, at lave ild med pinde. I troen på, "
+"at er en gud, giver aberne dig deres lavasten."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:9107]: Right...

File: files/lang/dk.po [hunk 5]
```diff
@@ -8934,6 +8999,9 @@ msgid ""
 "gives you a \"magic\" gold watch. Unbeknownst to him, the watch really is "
 "magical."
 msgstr ""
+"Som belønning, for at have hjulpet sin vogn op af en grøft, giver en "
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:9002]: I remember thinking about this... And I chose poorly. You are right.

File: files/lang/dk.po [hunk 6]
```diff
@@ -1710,8 +1710,7 @@ msgstr "Byggeri umuligt. Du har allerede bygget her i dag."
 msgid "For this action it is necessary to build a castle first."
 msgstr "Til denne handling er det nødvendigt først at bygge en borg."
 
-#, fuzzy
-msgid "Cannot build %{name} because the castle is too far away from an ocean."
+msgid "Cannot build %{name} because castle is too far from water."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po]: I'm not sure whether I did this right... I changed the string ending with "far from water" to "far away from ocean"

**Review by ihhub — APPROVED** [review]

**Pull request merged**